fe/projects/Account/webpack.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
         resetpasswordpage: ["./src/areas/Accounts/pages/ResetPassword/index.js", "./src/areas/Accounts/pages/ResetPassword/ResetPasswordPage.scss"],
         signinpage: ["./src/areas/Accounts/pages/SignIn/index.js", "./src/areas/Accounts/pages/SignIn/SignInPage.scss"],
         setpasswordpage: ["./src/areas/Accounts/pages/SetPassword/index.js", "./src/areas/Accounts/pages/SetPassword/SetPasswordPage.scss"],
-        contentpage: ["./src/areas/Home/pages/Content/index.js", "./src/areas/Home/pages/Content/ContentPage.scss"]
+        contentpage: ["./src/areas/Home/pages/Content/index.js", "./src/areas/Home/pages/Content/ContentPage.scss"],
```

Comment:
**question:** Is it used anywhere after changes?

Reply:
Yes, for regulations page.

Reply:
I would also apply the same to the regulations, because if someone doesn't want drop-down lists, they will simply complete the main text